component/http/middleware.go

```diff
@@ -2,6 +2,7 @@ package http
 
 import (
 	"errors"
+	"golang.org/x/time/rate"
```

Comment:
import order is still not correct. Can you check your IDE in order to make sure that yopu format on save?

Reply:
@mantzas  -  I think i raised this PR before making changes to IDE to autoformat. Let me fix this.